cf_xarray/accessor.py

```diff
@@ -354,7 +354,7 @@ def _get_with_standard_name(
         obj = obj._to_temp_dataset()
```

Comment:
```suggestion
        obj = obj.coords.to_dataset()
```

How about this?

Reply:
Oh, well yeah! Same result, but more elegant!